view/theme/frio/README.md

```diff
@@ -29,23 +29,23 @@ Don't blame me too much for ugly code and hacks. Fix it ;-)
 
 #### Screenshots
 **Default**
-![Default - Stream](https://github.com/rabuzarus/frio/blob/master/img/screenshots/screenshot.png)
+![Default - Stream](https://raw.githubusercontent.com/rabuzarus/frio/master/img/screenshots/screenshot.png)
```

Comment:
Are the images linked from @rabuzarus repository by purpose instead of the ~friendica repository?

Reply:
This is where they were referenced from, I didn't look further. Let me see if I can find them in the friendica repo.

Reply:
I found them on git.friendi.ca and updated the URLs.